docs/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+<!-- Thank you for submitting this issue!
+-->
+
+**Is this a BUG REPORT or FEATURE REQUEST?**:
+
+<!-- Provide a more detailed introduction to this issue and why it's relevant. 
+-->
+**Description**:
+
+<!-- Explain what you would expect from this process.
```

Comment:
I think instead of comments, Add Description so that new users would know what they have to answer the question.
If they want to answer they would delete the existing description and add their own answer.